index.html

```diff
@@ -4,6 +4,10 @@
 
 <head>
 
+<title>"Trago's Library</title>
+
+<link rel="short icon" type="image/jpg" href="T.jpg">
```

Comment:
isn't the rel supposed to be `shortcut icon` not `short`? also I think the general preference is to use a `.ico` for compatibility, specifically `favicon.ico`.

Reply:
Ah good catch! Sorry about that! Also I did not know about the .ico preference. I apologize.